src/Components/Taskbar/index.tsx

```diff
@@ -3,21 +3,36 @@ import { ICONS } from "../../Configs/Taskbar.config";
 import { ThemeStore, toggleTheme } from "../../store/darkMode";
 import styles from "./taskbar.module.css";
 import { TaskBarButton } from "./TaskbarButton";
+import startMenu from '../../assets/icons/taskbar/windows.png'
```

Comment:
The name of the image is case sensitive, we shall change it to `../../assets/icons/taskbar/Windows.png`

Reply:
haha, i missed that typo XD